Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -761,7 +761,7 @@ def _on_resume_err(self, failure):
         failure.trap(CancelledError)
         self._logger.error("Resume data cancelled")
 
-    @waitForHandleAndSynchronize()
+    @checkHandleAndSynchronize(default=Deferred().callback(None))
```

Comment:
Because `handle` must be valid as a precondition for `save_resume_data`. http://www.libtorrent.org/reference-Core.html#save_resume_data().

As for default value, I want to call all the callback related to the caller to write nothing. Because in this case the call is simply failed. (I was thinking to use `errback()`, but I think this is cleaner)